code/game/atoms.dm

```diff
@@ -254,14 +260,27 @@
 	if(article)
 		. = "[article] [src]"
 		override[EXAMINE_POSITION_ARTICLE] = article
+
 	if(SEND_SIGNAL(src, COMSIG_ATOM_GET_EXAMINE_NAME, user, override) & COMPONENT_EXNAME_CHANGED)
 		. = override.Join("")
 
+///Generate the full examine string of this atom (including icon for goonchat)
 /atom/proc/get_examine_string(mob/user, thats = FALSE)
 	. = "[icon2html(src, user)] [thats? "That's ":""][get_examine_name(user)]"
 
 /atom/proc/examine(mob/user)
-	to_chat(user, get_examine_string(user, TRUE))

```

Comment:
Leaving another comment as part of this review. The functionality of displaying blood as seen in this proc should be rolled into get_examine_string() and this proc rolled back to relying on get_examine_string(), as get_examine_string() is used multiple times elsewhere and has not been properly updated to reflect blood being inherent to the base atom type.

Reply:
It's literally only used for blood overlay name updating. There's nothing else that uses it currently at all.

Reply:
![image](https://user-images.githubusercontent.com/6356337/67255322-72600580-f44f-11e9-8ed5-7030e0e65694.png)

Reply:
And what does get_examine_string do, perchance?

Reply:
get_examine_string() generates the name for an item, allowing things like blood to show up when you're examining a spaceman without the copypasta associated with the explicit checks for blood. this also allows the examine string to be overridden by individual objects, though this behavior isn't yet used in tgcode. prior to the proc being implemented, every single one of those get_examine_string() calls had their own individual checks for blood, which was extremely messy.

Reply:
it's still checking each item for the component before the reversion. The current code updates its status the same way the component basically did, just with less obfuscation that the component brings. 

It still isn't used for anything else, and either doesn't apply at all how it was (which was the case quite frequently) or EVERYTHING was blood stained

Reply:
It doesn't check for an explicit component. It checks for all components, which is how components are designed to work; to have behavior be easily expandable without directly touching the relevant code. The proc just needs to be modified to take into account blood no longer being a component, that's all I'm asking.

Reply:
It already has, though? it works as intended right now.